### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ip-ilya**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
+    def calculate_washing_price(self, car: Car) -> float:
+        cost = car.comfort_class * (self.clean_power - car.clean_mark) * \
+            (self.average_rating / self.distance_from_city_center)
+
+        return round(cost, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def serve_cars(self, list_of_cars: list) -> int | float:
```
Comment (pavlejviki) [inline app/main.py:32]: Add additional type hint for list inner objects like list[YOUR_VALUE_TYPE]

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        cost = car.comfort_class * (self.clean_power - car.clean_mark) * \
```
Comment (pavlejviki) [inline app/main.py:24]: See checklist how to break your calculations correctly and use ()

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```
Comment (Polyakiv-Andrey) [inline app/main.py:2]: after the opening bracket, write all arguments on a new line, including the first one

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,47 @@
...
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
```
Comment (Polyakiv-Andrey) [inline app/main.py:12]: after the opening bracket, write all arguments on a new line, including the first one

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**